shiksha-migration/update-intersted-to-join.js

```
@@ -0,0 +1,102 @@
+
+            // Only UPDATE — never INSERT. Skip if user does not exist in destination.
+            const updateQuery = `
+        UPDATE public."Users"
+        SET "UserInterestedToJoin" = $2
+        WHERE "UserID" = $1
+      `;
+
+            const result = await destClient.query(updateQuery, [userId, rawValue]);
+
+            if (result.rowCount > 0) {
+                console.log(`[UPDATE] ✅ Updated UserID=${userId}  →  UserInterestedToJoin="${rawValue}"`);
```

ESLint

[error] Replace `················console.log(`[UPDATE]·✅·Updated·UserID=${userId}··→··UserInterestedToJoin="${rawValue}"`);` with `········console.log(⏎··········`[UPDATE]·✅·Updated·UserID=${userId}··→··UserInterestedToJoin="${rawValue}"`,`

(prettier/prettier)